src/main/java/org/parchmentmc/compass/tasks/GenerateExport.java

```diff
@@ -65,4 +83,76 @@ protected MappingDataContainer modifyData(MappingDataContainer container) throws
 
     @OutputFile
     public abstract RegularFileProperty getOutput();
+
+    @Input
+    public abstract Property<Boolean> getUseBlackstone();
+
+    @Nullable
+    @Internal
+    protected SourceMetadata getSourceMetadata() throws IOException {
+        if (getUseBlackstone().get() == Boolean.TRUE) {
```

Comment:
Do we actually need the `==` check here? Since `.get()` will return a `Boolean` which will be unboxed.

Reply:
Actually, no. I forgot that if a property is set to null, then the convention will kick in. Meaning this can never be null.